src/middlewares/user-authentication.js

```diff
@@ -3,7 +3,10 @@
 const jwt = require('jsonwebtoken');
 
 module.exports = (req, res, next) => {
-  const { token } = req.cookies;
+  let { token } = req.cookies;
+  if (req?.headers?.swaggerToken) {
+    token = req.headers.swaggerToken;
+  }
```

Comment:
I had to send the authentication token with a header because the `Cookie` or `Set-Cookie` headers are forbidden headers and the browser forbids swagger from setting them. 
You can refer to this [issue](https://github.com/swagger-api/swagger-js/issues/1163) that explains the problem, and this [MDN article](https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name) for forbidden headers list.